README.md

```diff
@@ -67,7 +67,7 @@ if let Ok(output) = application.render(&mapping, &mut renderer) {
 // by default application won't process widget tree if nothing was changed.
 // "change" is either any widget state change, or new message sent to any widget (messages
 // can be sent from application host, for example a mouse click, or from another widget).
-application.forced_process();
+application.forced_process(Default::default());
```

Comment:
Ah, I fixed it. I had just forgotten to re-generate the README. We should probably have CI generate the readme and make sure that the diff between the readme before and after it generated it is blank so that it makes sure we didn't forget to generate it.